lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -416,6 +424,12 @@ defmodule ExUnit.Case do
       raise "@describetag must be set inside describe/2 blocks"
     end
 
+    for attribute <- Module.get_attribute(module, :ex_unit_registered_describe_attributes),
+        value = Module.get_attribute(module, attribute),
+        value not in [[], nil] do
```

Comment:
I don't have a good way to tell if the attribute is accumulated here. The problem that introduces is that technically you could do:

``` elixir
ExUnit.Case.register_describe_attribute(__MODULE__, :describe_fixture)
@describe_fixture []
describe "a thing" do
  test "has fixture without error", context do
    assert context.registered.describe_fixture == []
  end
end
```

Reply:
Yeah. For now maybe the best option is to remove the check.